api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -0,0 +1,77 @@
+import random
+
+import factory.fuzzy
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.offers import factories as offers_factories
+from pcapi.core.providers.titelive_gtl import GTLS
+
+
+def save_test_cases_sandbox() -> None:
+    create_offers_with_gtls()
+
+
+def create_offers_with_gtls() -> None:
+    venue = offerers_factories.VenueFactory(
+        name="Librairie des GTls",
+        venueTypeCode=offerers_models.VenueTypeCode.BOOKSTORE,
+        isPermanent=True,
+        latitude=45.91967,
+        longitude=3.06504,
+        address="13 AVENUE BARADUC",
+        postalCode="63140",
+        city="CHATEL-GUYON",
+        departementCode="63",
+    )
+    create_offers_for_each_gtl_level_1(10, venue)
+    create_offers_with_gtl_id("01030000", 10, venue)  # littérature, Œuvres classiques
+    create_offers_with_gtl_id("01030100", 10, venue)  # littérature, Œuvres classiques, Antiquité
+    create_offers_with_gtl_id(
+        "01030102", 10, venue
+    )  # littérature, Œuvres classiques, Antiquité, Littérature grecque antique
+
+    # un librairie que pour des mangas
+
+    venue2 = offerers_factories.VenueFactory(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏plutot que `venue` et `venue2` autant leur donner un vrai nom

`librairie_gtl` et `librairie_manga` par exemple

low logaf